src/Take.Blip.Builder.UnitTests/Actions/ProcessHttpActionTests.cs

```diff
@@ -216,6 +216,42 @@ public async Task ProcessActionWithBotIdentifierHeaderShouldSucceed()
                     h => h.RequestUri.Equals(settings.Uri)), Arg.Any<CancellationToken>());
         }
 
+        [Fact]
+        public async Task ProcessActionTimeoutShouldSucceed()
+        {
+            // Arrange
+            var settings = new ProcessHttpSettings
+            {
+                Uri = new Uri("https://blip.ai"),
+                Method = HttpMethod.Post.ToString(),
+                Body = "{\"plan\":\"Premium\",\"details\":{\"address\": \"Rua X\"}}",
+                Headers = new Dictionary<string, string>()
+                {
+                    {"Content-Type", "application/json"},
+                    {"Authorization", "Key askçjdhaklsdghasklgdasd="}
+                },
+                ResponseBodyVariable = "httpResultBody",
+                ResponseStatusVariable = "httpResultStatus",
+
+            };
+
+            var target = GetTarget();
+
+            HttpClient.SendAsync(Arg.Any<HttpRequestMessage>(), Arg.Any<CancellationToken>())
+                .Returns(async x =>
+                {
+                    await Task.Delay(20001, x.Arg<CancellationToken>());
```

Comment:
Replace hard-coded time based integers with timespan

```suggestion
                    await Task.Delay(TimeSpan.FromSeconds(20), x.Arg<CancellationToken>());
```

Reply:
Changed to TimeSpan.FromSeconds(20).